pkg/gofr/container/datasources.go

```diff
@@ -62,7 +64,7 @@ type Cassandra interface {
 	//	   }
 	//	   users := []user{}
 	//	   err := c.Query(&users, "SELECT * FROM users")
-	Query(dest any, stmt string, values ...any) error
+	Query(ctx context.Context, dest any, stmt string, values ...any) error
```

Comment:
we are creating a new interface `CassandraWithContext` and deprecating this.. so why are we changing methods of this interface? I think this needs to be reverted in both `Cassandra` as well as `CassandraBatch`

Reply:
Ohh yes! Will revert the changes in `Cassandra` and  `CassandraBatch`